tests/test_fremdkostenposition.py

```diff
@@ -0,0 +1,115 @@
+from datetime import datetime, timezone
+from decimal import Decimal
+
+import pytest  # type:ignore[import]
+
+from bo4e.com.betrag import Betrag
+from bo4e.com.fremdkostenposition import Fremdkostenposition, FremdkostenpositionSchema
+from bo4e.com.preis import Preis
+from bo4e.com.preisstaffel import Preisstaffel
+from bo4e.enum.mengeneinheit import Mengeneinheit
+from bo4e.enum.preisstatus import Preisstatus
+from bo4e.enum.waehrungscode import Waehrungscode
+from bo4e.enum.waehrungseinheit import Waehrungseinheit
+from tests.serialization_helper import assert_serialization_roundtrip  # type:ignore[import]
+from tests.test_menge import example_menge  # type:ignore[import]
+from tests.test_sigmoidparameter import example_sigmoidparameter  # type:ignore[import]
+
+
+class TestPreisstaffel:
+    @pytest.mark.parametrize(
+        "fremdkostenposition, expected_json_dict",
+        [
+            pytest.param(
+                Fremdkostenposition(
+                    positionstitel="Mudders Preisstaffel",
+                    von=datetime(2013, 5, 1, tzinfo=timezone.utc),
+                    bis=datetime(2014, 5, 1, tzinfo=timezone.utc),
+                    artikelbezeichnung="Dei Mudder ihr Preisstaffel",
+                    link_preisblatt=None,
+                    zeitmenge=None,
+                    einzelpreis=Preis(
+                        wert=Decimal(3.50),
+                        einheit=Waehrungseinheit.EUR,
+                        bezugswert=Mengeneinheit.KWH,
+                        status=Preisstatus.ENDGUELTIG,
+                    ),
+                    betrag_kostenposition=Betrag(
+                        waehrung=Waehrungscode.EUR,
+                        wert=Decimal(12.5),
+                    ),
+                    menge=None,
+                    artikeldetail=None,
+                    marktpartnercode=None,
+                    marktpartnername=None,
+                    gebietcode_eic=None,
```

Comment:
Wenn du oben default=None noch in die Klasse einbaust bei den optionalen Attributen, dann musst du sie hier nicht mehr angeben. Löst vermutlich auch deine 13 required attributes unten.

Reply:
e71a46b93d0216a5a6fb5e9eb930bac1ee8697c5